CHANGELOG.rst

```diff
@@ -12,6 +12,7 @@ Changelog
 0.16.11
 -------
 - fix: ``sqlite://:memory:`` in Windows thrown ``OSError: [WinError 123]``
+- Support specified ``pk`` to ``bulk_create()``
```

Comment:
This is not specific enough. Try something like:

Support ``bulk_create()`` insertion of records with overridden primary key when the primary key is DB-generated

Reply:
Oh I see, really thank you for correction. :+1: